CHANGES.rst

```diff
@@ -21,6 +21,8 @@ Imviz
 - Catalog Search plugin now exposes a maximum sources limit for all catalogs and resolves an edge case
   when loading a catalog from a file that only contains one source. [#3337]
 
+- Catalog Search plugin now supports loading catalog data directly from files. [#3359]
```

Comment:
can we clarify somehow that this adds support for the additional columns (otherwise I worry we might get confused that this is when support for loading from file was first introduced at all)?